ramls/check-in.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Check-in operation",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "Unique ID (generated UUID) of the check-in operation",
+      "$ref": "raml-util/schemas/uuid.schema"
+    },
+    "occurredDateTime": {
+      "type": "string",
+      "description": "Date and time when the check-in has been occurred",
+      "format": "date-time"
+    },
+    "itemId": {
+      "type": "string",
+      "description": "Id of item that has checked in",
+      "$ref": "raml-util/schemas/uuid.schema"
+    },
+    "checkInServicePointId": {
+      "type": "string",
+      "description": "Check-in service point id",
+      "$ref": "raml-util/schemas/uuid.schema"
+    },
+    "performedByUserId": {
+      "type": "string",
+      "description": "Id of user that has performed the check-in operation",
+      "$ref": "raml-util/schemas/uuid.schema"
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "id",
```

Comment:
Can `id` be required, if this JSON schema is used for the request body of the POST to create the record?

Reply:
I decided to mark it is as required (actually PUT is not implemented now). 

There was discussion on an inventory PR that an endpoint has optional ID field, but at the same time it shares schema for POST and PUT and for PUT the `id` is required.

Reply:
I don't think FOLIO has a clear standard for this, which makes it confusing.

Usually, the POST endpoint allows for the defaulting of the `id`, is that done for this API?

Reply:
No, the endpoint does not have defaulting for id.

Reply:
I think we should be consistent and provide the ability to default the ID

(It really isn't needed for this API, though if once path were chosen, I'd pick always defaulting the id)